docs/data/charts/radar/DemoRadar.tsx

```diff
@@ -1,9 +1,39 @@
 import * as React from 'react';
 import Box from '@mui/material/Box';
+import { useTheme } from '@mui/material/styles';
 import ChartsUsageDemo from 'docsx/src/modules/components/ChartsUsageDemo';
 import { Unstable_RadarChart as RadarChart } from '@mui/x-charts/RadarChart';
 
 export default function DemoRadar() {
+  const theme = useTheme();
+
+  const getStripeColorFunction = {
+    default: undefined,
+    'two-tones':
+      theme.palette.mode === 'dark'
+        ? (index: number) =>
+            index % 2 === 0
+              ? (theme.vars || theme).palette.primary.dark
+              : (theme.vars || theme).palette.grey[300]
+        : (index: number) =>
+            index % 2 === 0
+              ? (theme.vars || theme).palette.primary.light
+              : (theme.vars || theme).palette.grey[800],
+    null: null,
+  };
```

Comment:
Not sure if it's just me, but in dark mode I think the stripes are too vibrant and distract a bit from the content of the chart: 

![image](https://github.com/user-attachments/assets/1ddd9f06-4a29-48ed-a814-cd99c5c81431)

Maybe a more pale grey would be better?